logging/cluster-logging-release-notes.adoc

```diff
@@ -8,235 +8,14 @@ toc::[]
 [id="openshift-logging-5-0-about-this-release"]
 == About this release
 
-(link:https://errata.devel.redhat.com/docs/show/66974[RHBA-2020:66974-04 Errata Advisory for Openshift Logging 5.0.0]) is now available. New features, changes, and known issues that pertain to {ProductName} 5.0 are included in this topic.
+The following advisories are available for {ProductName} 5.0:
+
+* link:https://errata.devel.redhat.com/docs/show/66974[RHBA-2020:66974-04 Errata Advisory for Openshift Logging 5.0.0]
 
 [id="openshift-logging-5-0-inclusive-language"]
 == Making open source more inclusive
 
 Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[Red Hat CTO Chris Wright’s message].
 
-[id="openshift-logging-5-0-new-features-and-enhancements"]
```

Comment:
@vikram-redhat  I pasted all this content to cluster-logging-release-notes-5.0.0.adoc